src/components/TreeView/types.ts

```diff
@@ -5,10 +5,13 @@ export interface INodeView<T> {
   isOpened: boolean
   isSelected: boolean
   data: T
+  parent: T
```

Comment:
:warning: Will write it here, but saw in many places already.

We don't need to know parent of smth at any point anywhere. Please, look carefully and ping me if we actually need parent property anywhere (except for nest/unnest, which doesn't need it either)